readme.txt

```diff
@@ -16,8 +16,6 @@ Automatically tweets the post title or custom message and a link to the post.
 
 **Disclaimer:** *TWITTER, TWEET, RETWEET and the Twitter logo are trademarks of Twitter, Inc. or its affiliates.*
 
-**Note:** Post types are automatically set to autoshare. Future versions of this plugin could allow this to be set manually.
```

Comment:
@jeffpaul since we have settings now, this can be removed?

Reply:
@thrijith yes, seems that we can remove that (both README.md and readme.txt assuming it exists in both files).